config/default/webhookcainjection_patch.yaml

```diff
@@ -0,0 +1,15 @@
+# This patch add annotation to admission webhook config
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  labels:
+    app.kubernetes.io/name: mutatingwebhookconfiguration
+    app.kubernetes.io/instance: mutating-webhook-configuration
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/created-by: codeflare-operator
+    app.kubernetes.io/part-of: codeflare-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: mutating-webhook-configuration
+  annotations:
+     service.beta.openshift.io/inject-cabundle: "true"
+     service.beta.openshift.io/serving-cert-secret-name: codeflare-operator-raycluster-webhook-cert
```

Comment:
Is it really needed?

Reply:
Yes, I meant to remove only `service.beta.openshift.io/serving-cert-secret-name`.